cmd/test.go

```diff
@@ -20,8 +20,15 @@ func newTestCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// testCmd represents the test command
 	var testCmd = &cobra.Command{
 		Use:   "test",
-		Short: "Test your project in the local Appsody environment",
-		Long:  `This starts a docker container for your project and runs your test in it.`,
+		Short: "Test your project in the local Appsody environment.",
+		Long: `Run the local Appsody environment, starting a container-based continuous build environment for your project, and running the test suite each time a file changes.
```

Comment:
Should have a comma after "container-based": 
"[...] a container-based, continuous build environment [...]"